src/drive/web/modules/navigation/duck/reducer.js

```diff
@@ -358,20 +358,17 @@ export const getFileById = ({ view }, id) => {
   return ensureFileHavePath({ view }, file)
 }
 
-const isRootFolder = folder => folder.id === ROOT_DIR_ID
-
 const ensureFileHavePath = ({ view }, file) => ({
   ...file,
   path: getFilePath({ view }, file)
 })
 
 export const getFilePath = ({ view }, file) => {
   const { displayedFolder } = view
-  return isDirectory(file)
+  const isRecentView = displayedFolder === null
+  return isDirectory(file) || isRecentView
     ? file.path
-    : displayedFolder && !isRootFolder(displayedFolder)
-      ? `${displayedFolder.path}/${file.name}`
-      : `/${file.name}`
+    : `${displayedFolder.path}/${file.name}`.replace('//', '/')
```

Comment:
Good to know, we can't create a folder or a file with an `/` in its name. And if we create a folder / file with an `/`, it'll be replaced by `:`.  So this regex seems to work even if I would prefer to check with `isRootFolder` since it more robust